### PR TITLE
Implement File Size Display on Downloads Page - Resolves #322

### LINKED ISSUE #322: [req] put size of file on the webpage please
It would be nice to show the size of the file on the webpage listing all the downloads...thanks

### PR DESCRIPTION
**Pull request by vkartk**
This PR addresses Issue #322 by adding the functionality to display file sizes on the downloads page. The changes have been implemented in both the backend and frontend of the application.

###   Backend:

- Updated API endpoints to return file size information.
- Implemented efficient file size calculation.

###     Frontend:

- Modified the downloads page to include file size display.
- Adjusted UI elements to accommodate file size information, ensuring readability and design consistency.

### CODE REVIEW

**Comment by vkartk** [comment]
image

File: app/ytdl.py [hunk 1]
```diff
@@ -138,7 +139,9 @@ async def update_status(self):
                 return
             self.tmpfilename = status.get('tmpfilename')
             if 'filename' in status:
-                self.info.filename = os.path.relpath(status.get('filename'), self.download_dir)
+                fileName = status['filename']
```
Comment (alexta69) [inline app/ytdl.py:142]: The reason it was `status.get('filename')` and not `status['filename']` is so we don't crash if the status doesn't contain the filename. Can you change the code to handle this case?

**Comment by alexta69** [comment]
Thanks @vkartk !

**Comment by vkartk** [comment]
@alexta69 Welcome :)

**Pull request merged**